client/post-editor/editor-sharing/publicize-options.jsx

```diff
@@ -152,15 +152,12 @@ module.exports = React.createClass( {
 		}
 
 		return (
-			<AddNewButton
-				isCompact={ true }
-				onClick={ this.newConnection }
-				>
-				{ this.translate( 'Connect new service' ) }
-			<span className="editor-sharing__external-link-indicator">
-				<Gridicon icon="external" size={ 18 } />
-			</span>
-			</AddNewButton>
+			<Button borderless compact={ true } onClick={ this.newConnection }>
```

Comment:
Minor: A bit of mixed conventions here `borderless compact={ true }`

Should be either `borderless={ true } compact={true }` or `borderless compact`

My vote is for the latter :smile: